hazelcast/src/main/java/com/hazelcast/sql/impl/type/converter/AbstractObjectConverter.java

```diff
@@ -33,8 +33,8 @@ protected AbstractObjectConverter(int id, QueryDataTypeFamily typeFamily) {
     }
 
     @Override
-    public boolean asBit(Object val) {
-        return resolveConverter(val, QueryDataTypeFamily.BOOLEAN).asBit(val);
+    public boolean asBoolean(Object val) {
+        return resolveConverter(val, QueryDataTypeFamily.BOOLEAN).asBoolean(val);
```

Comment:
It also makes sense to include changes to 01-type-system.md regarding BIT to BOOLEAN transition.

Reply:
Addressed.